client/signup/steps/upsell/form.jsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard, Button } from '@automattic/components';
+import { recordTracksEvent } from 'state/analytics/actions';
+import Main from 'components/main';
+
+/**
+ * Image dependencies
+ */
+import premiumThemesImage from 'assets/images/illustrations/themes.svg';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class UpsellForm extends Component {
+	static propTypes = {
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	handleSubmit = ( type ) => {
+		this.props.recordTracksEvent( 'calypso_signup_actions_submit_site_type', {
+			value: type,
+		} );
+		this.props.submitForm( type );
+	};
+
+	render() {
+		return (
+			<>
+				<Main className="upsell">
+					<CompactCard className="upsell__card-header">{ this.header() }</CompactCard>
+					<CompactCard className="upsell__card-body">{ this.body() }</CompactCard>
+					<CompactCard className="upsell__card-footer">{ this.footer() }</CompactCard>
+				</Main>
+			</>
+		);
+	}
+
+	header() {
+		return (
+			<header className="upsell__small-header">
+				<i className="upsell__title">
+					Your domain { this.props.domainName } is currently available
```

Comment:
Some more strings here that aren't wrapped in `translate()`

Reply:
Please check [comment](https://github.com/Automattic/wp-calypso/pull/40879#discussion_r414280361).